changelog/unreleased/pr-17925.toml

```diff
@@ -0,0 +1,9 @@
+type = "c"
+message = "Update com.github.zafarkhaja:java-semver from 0.9.0 to 0.10.2."
```

Comment:
We usually don't do changelog entries for dependency updates. Exceptions are security fixes.